src/components/Header/NavBar/NavBar.test.js

```diff
@@ -12,4 +13,14 @@ describe('<NavBar />', () => {
   it('should render component', () => {
     expect(wrapper).toMatchSnapshot();
   });
+
+  it('should render login if auth is false', () => {
+    const nav = wrapper.find(ConnectedAuthorization).prop('children')(false, jest.fn());
```

Comment:
There is no need to import `ConnectedAuthorization` component. Use string to identify component you want to find.
`wrapper.find('ConnectedAuthorization').prop('children')`